### PR TITLE
bpo-36277: Add document for pdb debug and retval commands

### PR DESCRIPTION
**Pull request by nguyendv**
https://bugs.python.org/issue36277

Automerge-Triggered-By: @csabella

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Our records indicate we have not received your CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.

If you have recently signed the CLA, please wait at least one business day
before our records are updated.


You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for your contribution, we look forward to reviewing it!

**Comment by nguyendv** [comment]
I'm not sure if a News entry is necessary. Please let me know if it's not.

**Comment by methane** [comment]
NEWS entries are for:

* Python users can see what's changed from previous release.
* Credit for the contribution.

If you don't want credit, I think we don't need NEWS entry for this change.

If you want credit, you should add "Patch by Dave Nguyen" at the end of the
NEWS entry, and add "Dave Nguyen" in Misc/ACKS file.

File: Doc/library/pdb.rst [hunk 1]
```diff
@@ -529,6 +529,14 @@ by the local file.
 
    Quit from the debugger.  The program being executed is aborted.
 
+.. pdbcommand:: debug code
+
+   Enter a recursive debugger that steps through the code
```
Comment (eamanu) [inline Doc/library/pdb.rst:534]: ```Enter in a recursive debugger...``` ?

**Review by eamanu — COMMENTED** [review]
IMO the news is not necessary here. But we have to wait for a core dev opinion

File: Misc/NEWS.d/next/Documentation/2019-04-19-21-45-18.bpo-36277.yVzAvJ.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Add document for pdb's debug and retval commands # Don't start with "- Issue
+#<n>: " or "- bpo-<n>: " or that sort of stuff.
+###########################################################################
```
Comment (remilapeyre) [inline Misc/NEWS.d/next/Documentation/2019-04-19-21-45-18.bpo-36277.yVzAvJ.rst]: This file should just read
```
Document pdb's debug and retval commands.
```
or
```
Document pdb's debug and retval commands. Patch contributed by ... .
```
Comments should be removed.

**Review by remilapeyre — CHANGES_REQUESTED** [review]

File: Doc/library/pdb.rst (hunk 1 above)
Comment (nguyendv) [reply Doc/library/pdb.rst:534]: This is a verbatim copy from docs string. I could be wrong here but "Enter in" doesn't sound better than "Enter"

File: Misc/NEWS.d/next/Documentation/2019-04-19-21-45-18.bpo-36277.yVzAvJ.rst (hunk 2 above)
Comment (nguyendv) [reply Misc/NEWS.d/next/Documentation/2019-04-19-21-45-18.bpo-36277.yVzAvJ.rst]: Thank you for the review. I've learned that I don't really need this file.

Comment (remilapeyre) [reply Misc/NEWS.d/next/Documentation/2019-04-19-21-45-18.bpo-36277.yVzAvJ.rst]: Yes you can remove it too, thanks for making the change :)

**Review by remilapeyre — APPROVED** [review]

File: Doc/library/pdb.rst (hunk 1 above)
Comment (JulienPalard) [reply Doc/library/pdb.rst:534]: @csabella can you please help us on this one? Thanks :)))

**Comment by JulienPalard** [comment]
(closing / reopening PR to restart failed Travis)

File: Doc/library/pdb.rst (hunk 1 above)
Comment (csabella) [reply Doc/library/pdb.rst:534]: `Enter` works here.

**Review by csabella — APPROVED** [review]
LGTM.  Thank you for the contribution, @nguyendv!

**Pull request merged**